helium/src/main/groovy/com/stanfy/helium/handler/tests/HttpExecutor.java

```diff
@@ -104,10 +107,17 @@ public HttpResponseWrapper performMethod(final Service service, final ServiceMet
     }
   }
 
-  private RequestBody getRequestBody(final ServiceMethod method, final ServiceMethodRequestValues request, final String encoding) {
+  private RequestBody getRequestBody(final ServiceMethod method, final ServiceMethodRequestValues request,
+                                     final String encoding, final Headers headers) {
     final RequestBodyBuilder builder = BuilderFactory.getBuilderFor(method.getBody());
-
-    return builder.build(types, request.getBody(), encoding);
+    MediaType contentType;
+    String headerContentType = headers.get("Content-Type");
```

Comment:
before this change helium was always setting `application/json` as a content type header for test requests, now we can override with our own header